website/blog/2024-07-12-thallium.md

```diff
@@ -0,0 +1,252 @@
+---
+author: Tomasz Godzik
+title: Metals v1.3.1 - Thallium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v1.3.3, which brings a number of
+long awaited bugfixes and some new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">40</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">36</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">3</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">15</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details:
+[https://github.com/scalameta/metals/milestone/69?closed=1](https://github.com/scalameta/metals/milestone/69?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs, Zed,
+Helix and Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Index implementations in JDK sources](#index-implementations-in-jdk-sources)
+- [Value completions for union types](#value-completions-for-union-types)
+- [Allow debugging for all build servers](#allow-debugging-for-all-build-servers)
+
+## Index implementations in JDK sources
+
+Since Metals v1.3.0 it's possible to search for implementations of classes or
+classes' members inside the dependency sources. However, this wasn't enabled for
+JDK sources as it added a substantial amount of indexing time even in smaller
+projects.
+
+In this version of Metals we added a central database that can be found under
+the `.metals` directory in user home, which contains shared indexes of JDK
+sources thus avoiding a need to index them more than once on any given machine.
+
+![jdk-sources](https://i.imgur.com/lTMmhD8.gif)
+
+This great improvement was added by [kasiaMarek](https://github.com/kasiaMarek)
+
+## Value completions for union types
+
+Union types are a new feature in Scala 3 that allows types to be combined when
+we need to specify that certain value can be of more than one type. If used
+together with intersection types it will still work as expected.
```

Comment:
Can you move this sentence `If used
 together with intersection types it will still work as expected.` to line 83. Otherwise it makes it sound like union types work with intersection, where we want to say that the feature also works with intersection.